components/org.wso2.micro.integrator.dashboard.web/web-app/src/home/GroupSelector.js

```
@@ -13,110 +13,121 @@ const ICP_NAME = window.icp.name;
+
+  const dispatch = useDispatch();
+  return <FormControl style={{ width: 150 }}>
+    <Select
+      classes={{ root: classes.selectRoot }}
+      value={selectedGroupId}
+      labelId="group-id-select-label"
+      id="group-id-select"
+      onChange={(e) => changeSelectedGroupId(e.target.value)}
+    >
+      {options.map((option) => (
+        <MenuItem value={option.value}>{option.label}</MenuItem>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)